test/integration/api_paths_test.go

```diff
@@ -27,6 +27,7 @@ func TestRootAPIPaths(t *testing.T) {
 		"/logs/": http.StatusNotFound,
 	}
 
+	defer testutil.RequireEtcd(t).Terminate(t)
```

Comment:
I wouldn't recommend this in normal code but in test code it's just there
to be good citizens

On Wed, Feb 24, 2016 at 6:39 PM, Clayton Coleman <ccoleman@redhat.com>
wrote:

> Yes
>
> On Wed, Feb 24, 2016 at 6:34 PM, Jordan Liggitt <notifications@github.com>
> wrote:
>
>> In test/integration/api_paths_test.go
>> <https://github.com/openshift/origin/pull/7286#discussion_r54026649>:
>>
>> > @@ -27,6 +27,7 @@ func TestRootAPIPaths(t *testing.T) {
>> >  		"/logs/": http.StatusNotFound,
>> >  	}
>> >
>> > +	defer testutil.RequireEtcd(t).Terminate(t)
>>
>> Does this evaluate the first function immediately, and the second
>> function deferred?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/openshift/origin/pull/7286/files#r54026649>.
>>
>
>